src/bp/ui-admin/src/Pages/Account/Login.tsx

```diff
@@ -73,15 +92,20 @@ export default class Login extends Component {
     }
   }
 
-  handleInputChange = e => this.setState({ [e.target.name]: e.target.value })
-  handleInputKeyPress = e => e.key === 'Enter' && this.login()
+  handleEmailChanged = e => this.setState({ email: e.target.value })
+  handlePasswordChanged = e => this.setState({ password: e.target.value })
+  handleKeyPressed = e => e.key === 'Enter' && this.login()
```

Comment:
Can we use native form onSubmit instead of handleKeyPressed and a onClick

concretely simply replace `<Fragment>` by `<form onSubmit={this.login}>`and remove onClick on the submit button